packages/mobile/src/send/reducers.ts

```diff
@@ -48,13 +57,32 @@ export const sendReducer = (
   }
 }
 
+const sendPaymentOrInvite = (state: State, amount: BigNumber) => {
+  const timestamp = Date.now()
```

Comment:
calling `now()` here would make the reducer not pure, do you think we can have `timestamp` as a part of `SEND_PAYMENT_OR_INVITE` action?

Reply:
That makes sense. I think Aaron suggested that but I forgot to implement it. Will do now and commit. Thanks for the review!